docs/site/src/components/AppRouter.js

```diff
@@ -47,168 +47,186 @@ const demos = requireDemos
     return res;
   }, []);
 
-export default function AppRouter() {
-  return (
-    <Router
-      history={hashHistory}
-      render={applyRouterMiddleware(useScroll())}
-    >
-      <Route title="Material UI" path="/" component={AppFrame}>
-        <IndexRoute dockDrawer component={Home} title={null} />
-        <Route
-          title="Getting Started"
-          path="/getting-started"
-          nav
-          component={AppContent}
-        >
-          <IndexRedirect to="installation" />
-          <Route
-            title="Installation"
-            path="/getting-started/installation"
-            content={requireDocs('./getting-started/installation.md')}
-            component={MarkdownDocs}
-            nav
-          />
-          <Route
-            title="Usage"
-            path="/getting-started/usage"
-            content={requireDocs('./getting-started/usage.md')}
-            component={MarkdownDocs}
-            nav
-          />
-          <Route
-            title="Server Rendering"
-            path="/getting-started/server-rendering"
-            content={requireDocs('./getting-started/server-rendering.md')}
-            component={MarkdownDocs}
-            nav
-          />
-          <Route
-            title="Examples"
-            path="/getting-started/examples"
-            content={requireDocs('./getting-started/examples.md')}
-            component={MarkdownDocs}
-            nav
-          />
-        </Route>
+export default class AppRouter extends Component {
+  static childContextTypes = {
```

Comment:
That would have been easier! 😅 

It's working as is, but I might revisit that later.